moped-editor/src/views/projects/projectView/ProjectSummary/SubprojectsTable.js

```diff
@@ -75,15 +77,15 @@ const SubprojectsTable = ({ projectId = null }) => {
       ),
     },
     {
-      title: "Current status",
-      field: "current_status",
+      title: "Status",
+      // todo: does the 'field' need to actually exist in the data?
+      field: "status",
```

Comment:
spoke to Chia about this - @johnclary change this comment to point out that `status` does not actually exist in the data, but that's ok-ish.